app/es_api/utils/queries.py

```diff
@@ -54,6 +54,28 @@ def add_search_filters(search, filters):
     return result_search
 
 
+def add_search_sort(search, filters):
+    """This helper method checks if one of the configured sort options was
+        sent with the request then adds it to the search query
+        search = Elasticsearch Search object
+        filters = object containing all the request parameters
+        returns -> modified Elasticsearch search object"""
+    # add sort if it was sent in the request
+    result_search = search
+
+    if 'sort' in filters:
+        key = filters['sort']
```

Comment:
Is there any sort of input validation that we need to be concerned with here?

Reply:
On line 68-72, we basically check that whatever field was sent over matches one of the configured keywords in the database. If not then we just ignore it.

Reply:
Makes sense thanks